src/kw/kw.cpp

```diff
@@ -8,16 +8,33 @@
  */
 
 #include <string>
+#include <ctime>
+
 using std::string;
 
 #include "mercury/kw/kw.h"
 
+#include <stdlib.h>
 #include <parson.h>
 #include <curl/curl.h>
 
 
+KanoWorld::KanoWorld(void)
+{
+    // Data file to store token and expiration date
+    data_filename = string(getenv("HOME")) + "/" + ".mercury_kw.json";
```

Comment:
Perhaps we could use something like `~/.cache/` or `~/.local/` to store all these files in the future

Reply:
Suggest we move this to a new changeset, still have so much to close this PR.